app/graphql/types/query_type.rb

```diff
@@ -73,17 +73,13 @@ def all_plans_by_user_id(user_id:, content_choice:)
     # Budget Details
     #
     #
-    field :all_details_by_plan_id, [Types::Custom::PlanDetailType], null: true do
+    field :all_plan_details, [Types::Custom::PlanDetailType], null: true do
```

Comment:
is the client using the field `all_details_by_plan_id `? Note that this will remove it and might break the client if they are relying on it.

Reply:
very good point small project but still completely unnecessary,  will revert back this change